gooddata-sdk/gooddata_sdk/catalog/data_source/declarative_model/data_source.py

```diff
@@ -35,6 +36,19 @@ def to_api(self, credentials: dict[str, Any] = None) -> DeclarativeDataSources:
                 data_sources.append(data_source.to_api())
         return DeclarativeDataSources(data_sources=data_sources)
 
+    @classmethod
+    def from_dict(cls, data: dict[str, Any], js_ids: bool = True) -> CatalogDeclarativeDataSources:
```

Comment:
js_ids is quite a cryptic name. 
What about "openApiKeys" or "camelCaseKeys"?
Any thoughts here, @pcerny ?

Reply:
Well, the name should be definitely in snake case :)